app/routes/changes-enquiries/award/functions.js

```diff
@@ -12,7 +12,7 @@ const secondaryNavigationList = secondaryNavigationHelper.navigationItems(active
 
 async function getAwardList(req, res) {
   try {
-    const award = await dataStore.cacheRetriveAndStore(req, null, 'awardDetails', () => {
```

Comment:
Ahh...a good example of my ability to spell

Reply:
I'm not the best like.